server/http/gateway/static.go

```diff
@@ -12,16 +14,19 @@ import (
 
 func (g *gateway) static(w http.ResponseWriter, req *http.Request, _, _ string) {
 	appID := appIDVar(req)
-
 	if appID == "" {
 		httputils.WriteError(w, utils.NewInvalidError("app_id not specified"))
 		return
 	}
 
 	vars := mux.Vars(req)
-	assetName := vars["name"]
+	assetName := path.Clean(vars["name"])
+	if assetName == "." || strings.Contains(assetName, "/..") {
```

Comment:
wouldn't `path.Clean` remove the "/.." ? https://play.golang.org/p/Vnfn7OawTG8

Reply:
Not always (see https://play.golang.org/p/WPI-alRj-Cn) but there was a typo, and it should just be a prefix check anyway. Done.